index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+
+
+	<article class="tops">
+		<header id="header" class="ones"><h1>Walri</h1></header>
+		<div>
+		<div><img src="images/walri.jpg" alt="a single walrus" height="168px" width="188px"></div>
+		<div><p class="words">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+		tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+		quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+		consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+		cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+		proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p></div>
+		<div class="one"><p>Sold Out</p></div>
```

Comment:
If you maintain proper indentation it'll make it easier to tell what elements are parents/children of each other. It's also just a good habit to get into